lib/ckb/transaction_size.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+module CKB
+  module TransactionSize
+    SERIALIZED_OFFSET_BYTESIZE = 4
+
+    class << self
+      def base_size
+        68 + SERIALIZED_OFFSET_BYTESIZE
+      end
+
+      def every_cell_deps
```

Comment:
`cell_dep_size` may be better?

Reply:
It's every `cell_dep` in tx size, cell_dep size + extra 4 bytes

Reply:
Why `cell_dep_size` and `header_dep_size` are fixed value?

Reply:
`struct`, all data fields are fixed.

Reply:
it’s a bit strange to add a plural form after `every` and `cell_deps` and `header_deps` are field in `transaction`.

Reply:
updated